packages/core/legos/package.json

```diff
@@ -85,6 +86,7 @@
     "@chainlink/intrinio-adapter": "*",
     "@chainlink/json-rpc-adapter": "*",
     "@chainlink/kaiko-adapter": "*",
+    "@chainlink/layer2-sequencer-health-adapter": "*",
```

Comment:
Surprised this one didn't make it through until now :man_shrugging:

Reply:
done!